sql/mysqld.cc

```diff
@@ -4079,8 +4079,13 @@ static int init_common_variables()
   {
     uint files, wanted_files, max_open_files;
 
+#if defined(HAVE_POOL_OF_THREADS)
+    wanted_files= threadpool_size;
+#else
+    wanted_files= 0;
+#endif
     /* MyISAM requires two file handles per table. */
-    wanted_files= (10 + max_connections + extra_max_connections +
+    wanted_files+= (10 + max_connections + extra_max_connections +
                    tc_size * 2);
```

Comment:
Why not:
```
#if defined(HAVE_POOL_OF_THREADS)
wanted_files+= threadpool_size;
#endif
```

Reply:
quite right. definitely looks better. Amended and PR updated.